gitpod-io/gitpod
# fix: Loading Spinner Alignment

## Description
Fixes Loading Spinner Alignments

## Related Issue(s)

Fixes #11696

## How to test

- Open Preview
- Open Teams Usage 
- See Loading Spinner & text change :eyes:

## Release Notes

```release-note
NONE
```

## Werft options:

      If enabled this will build `install/preview`
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
